frontend/src/pages/booking/components/BookingForm/BookingForm.tsx

```diff
@@ -0,0 +1,112 @@
+import { useState } from 'react';
+
+import styled from '@emotion/styled';
+
+import Input from '../../../../common/components/Input/Input';
+import BookingSummarySection from '../BookingSummarySection/BookingSummarySection';
+import FormField from '../FormField/FormField';
+
+function BookingForm() {
+  const [menteeName, setMenteeName] = useState('');
```

Comment:
p4: 기본 시작 값이 매직넘버인데 상수화 하는 건 어떨까? 🤔 

```suggestion
const INITIAL_MENTEE_NAME = ''
  const [menteeName, setMenteeName] = useState(INITIAL_MENTEE_NAME);
```

Reply:
상수화를 하는 이유는 일반적으로 변할 수 있는 값을 상수로 분리해두어 변경에 유연하게 대응하기 위함이라고 생각해

그런데 이 경우에는 초기값이 단순히 빈 문자열일 뿐이고, 다른 값으로 변경될 가능성도 없는 고정된 타입의 기본값이라 굳이 상수로 분리하지 않아도 될 것 같아!

Reply:
값이 변경될 일이 적어서 상수로 분리하지 않아도 될 거 같다는 의견 좋네!! 👍👍
상수로 해두면 가독성 차원에서 좋을 수 있다고 생각했는데 데이지 말대로 굳이 할 필요는 없을 것 같네 ㅎㅎ 
좋은 의견 고마워~~